### PR TITLE
Fix test mps key search in blocklist dicts

### PR DESCRIPTION
**Pull request by DenisVieriu97**

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -9806,7 +9806,7 @@ def get_error_message(self, key, op_name, dtype):
             return f"Running test with {op_name} expected to fail due to unsupported MPS data type so skipping"
         elif key in self.UNIMPLEMENTED_OPS and dtype in self.UNIMPLEMENTED_OPS[key]:
             return f"Running test with {op_name} expected to fail due to missing op implementation"
-        return f"Running test with {op_name} hangs so skipping"
+        return False
```
Comment (razarmehr) [inline test/test_mps.py:9809]: Can you return None here?

**Review by razarmehr — CHANGES_REQUESTED** [review]

File: test/test_mps.py (hunk 1 above)
Comment (DenisVieriu97) [reply test/test_mps.py]: Thanks. Updated

**Review by Ronian526 — APPROVED** [review]
Looks good, thanks Denis.

**Review by razarmehr — APPROVED** [review]
LGTM

**Pull request merged**